zambeze/orchestration/plugin_modules/globus/globus_common.py

```diff
@@ -0,0 +1,390 @@
+# Local imports
+from ...identity import validUUID
+
+# Standard imports
+import os
+from os.path import basename
+from os.path import dirname
+from os.path import exists
+
+import re
+
+
+def localEndpointExists(globus_uuid: str, endpoint_list: list[dict]) -> str:
+    for item in endpoint_list:
+        if item["uuid"].lower() == globus_uuid.lower():
+            return True
+    return False
+
+
+def localEndpointConfigCheck(config: dict):
+    # Check that the UUIDs are correct
+    if "local_endpoints" in config:
```

Comment:
These would be better to be a constant. If for some reason we decide to change the name of the field in the future, we would just change in the constant value.

Reply:
I am not a Python expert, can you explain a little bit more about how to make things constant? I'm not sure I know how to do that in Python? (I miss C++)

Maybe something like this?

```python
def constant(f):
    def fset(self, value):
        raise TypeError
    def fget(self):
        return f()
    return property(fget, fset)

class _Const(object):
    @constant
    def FOO():
        return 0xBAADFACE
    @constant
    def BAR():
        return 0xDEADBEEF

CONST = _Const()

print(hex(CONST.FOO))  # -> '0xbaadfaceL'
```

So instead of passing in a dict you are passing in a specific class?

Reply:
This is another general comment for future improvements. No need to change anything now.

Unfortunately constants in Python are not like in C, as they can be changed during execution... So what we do is just to create a typical Python variable but with a value assigned to it, and use it in capital letters to let you and other developers know that it should be a constant. E.g..,:
```
PI = 3.14159265
```

What I meant here is for us to try to use a "controlled vocabulary" separate file or class that would have all these schema fields. Then, if we decide to change the field name in the future and many other parts of the code use that field, we would only change in that file, as the other parts in the code would be using the constant. I can clarify later in a call.